include/bluetooth/mesh/light_xyl_srv.h

```diff
@@ -0,0 +1,267 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/**
+ * @file
+ * @defgroup bt_mesh_light_xyl_srv Light xyL Server model
+ * @{
+ * @brief API for the Light xyL Server model.
+ */
+
+#ifndef BT_MESH_LIGHT_XYL_SRV_H__
+#define BT_MESH_LIGHT_XYL_SRV_H__
+
+#include <bluetooth/mesh/light_xyl.h>
+#include <bluetooth/mesh/model_types.h>
+#include <bluetooth/mesh/lightness_srv.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct bt_mesh_light_xyl_srv;
+
+/** @def BT_MESH_LIGHT_XYL_SRV_INIT
+ *
+ * @brief initialization parameters for a @ref bt_mesh_light_xyl_srv instance.
+ *
+ * @param[in] _lightness_handlers Lightness server callbacks.
+ * @param[in] _light_xyl_handlers Light xyL server callbacks.
+ */
+#define BT_MESH_LIGHT_XYL_SRV_INIT(_lightness_handlers, _light_xyl_handlers)   \
+	{                                                                      \
+		.handlers = _light_xyl_handlers,                               \
+		.lightness_srv =                                               \
+			BT_MESH_LIGHTNESS_SRV_INIT(_lightness_handlers),       \
+		.pub = { .msg = NET_BUF_SIMPLE(BT_MESH_MODEL_BUF_LEN(          \
+				 BT_MESH_LIGHT_XYL_OP_RANGE_STATUS,            \
+				 BT_MESH_LIGHT_XYL_MSG_LEN_RANGE_STATUS)) },   \
+		.range = {                                                     \
+			.min = { .x = 0, .y = 0 },                             \
+			.max = { .x = UINT16_MAX, .y = UINT16_MAX }            \
+		}                                                              \
+	}
+
+/** @def BT_MESH_MODEL_LIGHT_XYL_SRV
+ *
+ * @brief Light XYL Server model composition data entry.
+ *
+ * @param[in] _srv Pointer to a @ref bt_mesh_light_xyl_srv instance.
+ */
+#define BT_MESH_MODEL_LIGHT_XYL_SRV(_srv)                                      \
+	BT_MESH_MODEL_LIGHTNESS_SRV(&(_srv)->lightness_srv),                   \
+		BT_MESH_MODEL_CB(BT_MESH_MODEL_ID_LIGHT_XYL_SRV,               \
+				 _bt_mesh_light_xyl_srv_op, &(_srv)->pub,      \
+				 BT_MESH_MODEL_USER_DATA(                      \
+					 struct bt_mesh_light_xyl_srv, _srv),  \
+				 &_bt_mesh_light_xyl_srv_cb),                  \
+		BT_MESH_MODEL_CB(BT_MESH_MODEL_ID_LIGHT_XYL_SETUP_SRV,         \
+				 _bt_mesh_light_xyl_setup_srv_op,              \
+				 &(_srv)->setup_pub,                           \
+				 BT_MESH_MODEL_USER_DATA(                      \
+					 struct bt_mesh_light_xyl_srv, _srv),  \
+				 NULL)
```

Comment:
The changes look fine to me, but the build is failing. Please take a look at the model listings in the test. Also, this setup_pub isn't initialized with a buffer. It should just be removed IMO.